Cargo.toml

```diff
@@ -98,98 +98,95 @@ massa_versioning = { path = "./massa-versioning" }
 massa_wallet = { path = "./massa-wallet" }
 
 # Massa projects dependencies
-massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "branch" = "main"}
-massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "branch" = "main"}
+massa-proto-rs = {git = "https://github.com/massalabs/massa-proto-rs", "rev" = "cc0b0b3d9b245ac517d28ac6dcf1e87d72b881b7"}
+massa-sc-runtime = {git = "https://github.com/massalabs/massa-sc-runtime", "rev" = "a50d690ec5f7c3d4267e61c6d283e55dff49348f"}
 peernet = { git = "https://github.com/massalabs/PeerNet", "branch" = "main" }
 
 # Common dependencies
 transition = { git = "https://github.com/massalabs/transition.git", "rev" = "93fa3bf82f9f5ff421c78536879b7fd1b948ca75" }
-unsigned-varint = { version = "0.7.1", "git" = "https://github.com/cyphar/unsigned-varint.git", "branch" = "nom6-errors" }
+unsigned-varint = { version = "=0.7", "git" = "https://github.com/cyphar/unsigned-varint.git", "branch" = "nom6-errors" }
 substruct = { git = "https://github.com/sydhds/substruct", branch = "main" }
 machine = { git = "https://github.com/antifuchs/machine", "branch" = "fix-workspace-build" }
 aes-gcm = "0.10"
 anyhow = "1.0"
 assert_matches = "1.5"
-async-trait = "0.1.58"
+async-trait = "0.1"
 atty = "0.2"
 bitvec = "1.0"
-blake3 = "=1.3"
-bs58 = "=0.4"
+blake3 = "=1.4"
+bs58 = "=0.5"
 config = "0.13"
 console = "0.15"
-criterion = "0.4"
+criterion = "0.5"
 crossbeam = "0.8"
 crossbeam-channel = "0.5"
-ctrlc = "3.2.5"
```

Comment:
Is this backport made on purpose ?

Reply:
it's not a backport: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html